src/main/kotlin/com/expedia/graphql/extensions/GraphQLSchemaExtensions.kt

```diff
@@ -0,0 +1,32 @@
+package com.expedia.graphql.extensions
+
+import graphql.schema.GraphQLSchema
+import graphql.schema.idl.SchemaPrinter
+
+fun GraphQLSchema.print(
+    includeScalarTypes: Boolean = true,
+    includeExtendedScalarTypes: Boolean = true,
+    includeIntrospectionTypes: Boolean = true,
+    includeSchemaDefinition: Boolean = true,
+    includeDirectives: Boolean = true
+): String {
+    val schemaPrinter = SchemaPrinter(
+        SchemaPrinter.Options.defaultOptions()
+            .includeScalarTypes(includeScalarTypes)
+            .includeExtendedScalarTypes(includeExtendedScalarTypes)
+            .includeIntrospectionTypes(includeIntrospectionTypes)
+            .includeSchemaDefintion(includeSchemaDefinition)
+            .includeDirectives(false)
```

Comment:
ugh.... this flag removes both directive definitions as well as their target locations :(

Reply:
Added an issue here: https://github.com/graphql-java/graphql-java/issues/1587